src/utils/ucc_parser.c

```diff
@@ -589,18 +606,18 @@ ucc_status_t ucc_add_team_sections(void                *team_cfg,
         sec_name = kh_key(sections, i);
         sec      = kh_val(sections, i);
         if (ucc_check_section(sec->desc, vendor, model, team_size,
-                              ppn_min, ppn_max, nnodes)) {
+                              ppn_min, ppn_max, sock_min, sock_max, nnodes)) {
             sec_h = &sec->vals_h;
             j = kh_get(ucc_sec, sec_h, tune_key);
             if (j != kh_end(sec_h)) {
                 *tuning_str = kh_val(sec_h, j);
             }
             status = ucc_apply_file_cfg(team_cfg, tl_fields, env_prefix,
                                         component_prefix, sec_name);
-            return status;
+            found = 1;
```

Comment:
This is to allow the conf file to match multiple sections and take whatever section is matched last. The idea is that in the conf, the sections can be sorted from generic to specific going from top of the file to bottom. This allows a generic Grace config in case the more specific configs (say 2 socket 72 procs/socket) don't match